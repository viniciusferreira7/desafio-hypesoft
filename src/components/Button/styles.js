import styled, { css } from 'styled-components';

export const Container = styled.button`
${({ theme, background, sizes }) => css`
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  padding: 1.2rem ${theme.spacings.large};
  letter-spacing: 0.02rem;
  text-transform: uppercase;
  background:${background ? theme.colors.secondaryColor : theme.colors.white};
  color:${background ? theme.colors.white : theme.colors.primaryColor};
  font-size: ${sizes}rem;
  border-radius: 5px;
  border: none;
  transition: all 500ms ease-in-out;
  cursor: pointer;

&:hover {
  background:${!background ? theme.colors.secondaryColor : theme.colors.white};
  color:${!background ? theme.colors.white : theme.colors.primaryColor};
}
`}
`;
