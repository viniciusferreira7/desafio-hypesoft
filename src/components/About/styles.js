import styled, { css } from 'styled-components';
import { Container as Button } from '../Button/styles';
import { Container as TextContainer } from '../TextComponent/styles';

export const Container = styled.div`
${() => css`
  width: 100%;
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-column-gap: 2.8rem;
  grid-template-areas:'SrcImg' 'Description';
  padding-top: 10rem;
`}
`;

export const SrcImg = styled.div`
${({ theme, srcImg }) => css`
  grid-area: 'SrcImg';
  max-width: 56rem;
  width: 100%;
  height: 54.2rem;
  background-image: url(${srcImg});
  background-size: cover;
  border-radius: 1rem;
  box-shadow: 3.2rem 3.2rem   ${theme.colors.white};

  @media ${theme.media.lteMedium}{
    display: none;
  }
`}
`;

export const Description = styled.div`
${() => css`
  grid-area: 'Description';
  max-width: 67.2rem;

  > ${Button} {
    width: 16.2rem;
    height: 5.6rem;

    > img{
      margin-left: 1.1rem;
    }
  }
`}
`;

export const InfoHeading = styled.div`
${() => css`
`}
`;

export const InfoContainer = styled.div`
${({ theme }) => css`
  max-width: 67.2rem;
  display: flex;

  @media ${theme.media.lteMedium}{
    flex-direction: column;
    padding-bottom: 3.2rem;
  }
`}
`;

export const Group = styled.div`
${() => css`
  max-width: 20.6rem;
  display: flex;
  align-items: center;
  margin-right: 5rem;

  > ${TextContainer}:nth-child(2){
    max-width: 9.5rem;
    margin-left: 1.6rem;
    padding: 0;
  }
`}
`;

`}`;

export const GroupTwo = styled.div`
${() => css`
  width: 25.1rem;
  display: flex;
  align-items: center;

  > ${TextContainer}:nth-child(2){
    max-width: 9.5rem;
    margin-left: 1.6rem;
    padding: 0;
  }
`}
`;

`}`;
