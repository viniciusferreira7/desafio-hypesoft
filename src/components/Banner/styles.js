import styled, { css } from 'styled-components';
import { Title as Heading } from '../Heading/styles';
import { Container as Button } from '../Button/styles';
import { Container as GridDot } from '../GridDot/styles';

export const Container = styled.section`
${({ theme }) => css`
  width: 100%;
  max-height:31.2rem;
  overflow: hidden;
  padding: 6.4rem 2%;
  position: relative;
  margin-top: 16.4rem;

  @media ${theme.media.lteMedium}{
      max-height:auto;
      padding-top: 3rem
    }

 >  ${GridDot}{
    opacity: 0.15;
    position: absolute;
    top: 6.4rem;
    left: 12.2rem;
  }


  ${Heading}{
      width:50rem;
      height: 9.6rem;
      margin: 0 auto;


      @media ${theme.media.lteMedium}{
        width: 100%;
        text-align: center;
        height: 15rem;
        margin-bottom: 7rem;
      }

      > span{
        color:#bc199c;
      }

      > span::before{
        display: none;
      }
    }

  > ${Button} {
      width: 17.7rem;
      height: 5.6rem;
      margin: 0 auto;
      margin-top: 7.2rem;

      > a{
        margin-right: 0.9rem;
      }
  }
`}
`;

export const SmallEllipse = styled.div`
  ${({ theme }) => css`
    width:7.6rem;
    height:7.6rem;
    border-radius: calc(7.6rem / 2);
    background: ${theme.colors.white};
    opacity: 0.03;
    position: absolute;
    top: 2.6rem;
    right: 11.6rem;

    @media ${theme.media.lteMedium}{
      top: 8rem;
      right: 0;
    }
  `}
`;

export const HugeEllipse = styled.div`
  ${({ theme }) => css`
    width:52.9rem;
    height:52.9rem;
    border-radius: calc(52.9rem / 2);
    background: ${theme.colors.white};
    opacity: 0.03;
    position: absolute;
    top: 13.9rem;
    right: 90.7rem;

    @media ${theme.media.lteMedium}{
      width:30rem;
      height:30rem;
      top: 0;
      right: 4rem;
    }
  `}
`;

export const ContainerGridDot = styled.div`
  ${() => css`
    position: absolute;
    top:15rem;
    right: 21rem;
    transform: rotate(-90deg);

    ${GridDot}{
      opacity: 0.3;

    }
  `}
`;
