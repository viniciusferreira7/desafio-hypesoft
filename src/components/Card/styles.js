import styled, { css } from 'styled-components';
import { Container as TextContainer } from '../TextComponent/styles';

export const Container = styled.div`
${({ theme }) => css`
  background: linear-gradient(0deg, rgba(223, 227, 235, 0.4), rgba(223, 227, 235, 0.4)), #FFFFFF;
  border-radius: 6px;
  width: 31.2rem;
  max-height: 35.1rem;
  transition: all 400ms ease-in-out;

  @media ${theme.media.lteMedium}{
    width: 18.5rem;
    padding-bottom: 2%;
    max-height: 40rem;

    > svg {
      width:2.5rem !important;
      height:2.5rem !important;
      margin: 0;
    }

    ${TextContainer}{
      text-align: center;
      font-size: 2rem;
      padding:2.2rem 0 3.2rem 0 !important ;

      > strong {
        margin-top: 1.6rem;
      }
    }

  }

    > svg{
      width:4rem;
      height:4rem;
      margin: 2.2rem 2.2rem 0 2.2rem;
      color: #242424;
    }

    ${TextContainer}{
      padding:2.2rem 0 3.2rem 2.8rem ;
      position: relative;

      > strong {
        width: 6.4rem;
        height: 0.3rem;
        background-color: #ED1CA6;
        margin-top: 1.6rem;
        margin-bottom: 2.8rem;
        position: absolute;
        bottom: -1.3rem;
        left:6rem ;
      }
    }

    > img{
      margin: 2.8rem 0 3.6rem 3.5rem;
    }


    &:hover{
      background: #FFFFFF;
      box-shadow: 0px 20px 48px rgba(10, 25, 49, 0.09);
      border-radius: 6px;
      border: 0.3rem solid #ED1CA6;

      ${TextContainer}{
        color:#ED1CA6 ;
      }

      > svg {
        color: #ED1CA6;
      }
    }

;
`}
`;
