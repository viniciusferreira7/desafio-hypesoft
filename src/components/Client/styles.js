import styled, { css } from 'styled-components';
import { Container as TextComponent } from '../TextComponent/styles';

export const Container = styled.div`
${() => css`
  display: flex;
  align-items: center;
  width: 31.2rem;
  height: 10rem;
  background: linear-gradient(0deg, rgba(223, 227, 235, 0.3), rgba(223, 227, 235, 0.3)), #FFFFFF;
  border-radius: 0.4rem;
`}
`;

export const Photo = styled.div`
${({ theme, srcImg }) => css`
  width: 5.2rem;
  height: 5.2rem;
  border-radius: calc(5.2rem / 2);
  background-image:url(${srcImg});
  background-size: cover;
  background-color: ${theme.colors.gray_500};
  margin: 2.4rem;
  `}
`;

export const Info = styled.div`
${() => css`
  ${TextComponent}{
    text-transform: capitalize;
    padding: 0;
    padding-bottom: 0.6rem;

    > span{
      color: #ED1CA6
;
    }
  }
`}
`;
