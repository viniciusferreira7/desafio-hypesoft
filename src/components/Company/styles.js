import styled, { css } from 'styled-components';
import { Title as Heading } from '../Heading/styles';

export const Container = styled.div`
${({ theme }) => css`
  width: 100%;
  display: flex;
  padding-top: 20rem;
  justify-content: space-around;

  @media ${theme.media.lteMedium}{
    flex-direction: column;
    text-align: center;
  }

  ${Heading}{
    width: 312px;

    > strong{
      color:#bc199c;
      font-weight: 400;
    }
  }
`}
`;

export const ContainerLogo = styled.div`
${({ theme }) => css`
  display: flex;
  max-width: 93.6rem;
  max-height: 12.0rem;
  justify-content: space-around;

  @media ${theme.media.lteMedium}{
    flex-direction: column;
    width: 100%;
    max-height: 96.6rem;
  }
`}
`;
export const Logo = styled.div`
${({ theme }) => css`
  display: flex;
  align-items: center;
  justify-content: center;
  width: 168px;
  height: 120px;
  margin-right: 2.4rem;
  background: linear-gradient(0deg, rgba(223, 227, 235, 0.3), rgba(223, 227, 235, 0.3)), #FFFFFF;
  border-radius: 0.8rem;
  transition: all 500ms ease-in-out;

  @media ${theme.media.lteMedium}{
    margin: 0 auto;
    margin-top: 3rem;
  }

  &:hover{
    border:1rem solid ${theme.colors.white};
  }

  >img{
    width: 11.304rem;
    height: 2.6rem;
  }

  >img:nth-of-type(5){
    width:2.756rem !important;
    height:1.205rem !important;
  }

;

`}
`;
