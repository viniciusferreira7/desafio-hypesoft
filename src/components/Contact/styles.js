import styled, { css } from 'styled-components';
import { Container as GridDot } from '../GridDot/styles';

export const Container = styled.section`
${({ theme }) => css`
  width: 100%;
  display: flex;
  padding: 10rem;
  align-items: center;
  margin-top: 22.7rem;
  position: relative;

  ${GridDot}{
    position: absolute;
    top: 16%;
    right:12%;
    transform: rotate(-90deg);
    opacity: 0.3;

    @media ${theme.media.lteMedium}{
      display: none;
  }
  }

  @media ${theme.media.lteMedium}{
    padding: 2rem;
    flex-direction: column;
  }
`}`;
