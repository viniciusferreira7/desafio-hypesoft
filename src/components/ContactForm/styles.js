import styled, { css } from 'styled-components';
import { Container as Button } from '../Button/styles';

export const Container = styled.form`
${({ theme }) => css`
  width: 100%;
  max-width: 76rem;
  padding: 4.8rem;
  background: ${theme.colors.white};
  box-shadow: 0rem 1.2rem 2.4rem rgba(10, 25, 49, 0.02);
  border-radius: 0.8rem;

  @media ${theme.media.lteMedium}{
    padding: 2rem;
  }
  `}
`;
export const GroupInput = styled.form`
${({ theme }) => css`
  width: 100%;
  max-width: 66.4rem;
  min-height: 28.8rem;
  margin: 2.4rem 0 3.2rem 0;

  input{
    width: 100%;
    padding: 1.4rem 2rem;
    background: ${theme.colors.white};
    border: 0.1rem solid ${theme.colors.gray_100};
    color:${theme.colors.gray_500};
    border-radius: 0.4rem;
    outline: none;
    margin-bottom: 1.6rem;
    transition: all 300ms ease-in-out;

    &:focus{
      border: 0.2rem solid ${theme.colors.secondaryColor};
    }

    &:first-of-type{
      margin-right: 1.6rem;

      @media ${theme.media.lteMedium}{
        margin-right: 0;
      }
    }
  }

  textarea{
    padding: 1.4rem 2.4rem;
    max-width: 66.4rem;
    width: 100%;
    height: 15.2rem;
    resize: none;
    outline: none;
    border: 0.1rem solid ${theme.colors.gray_100};
    transition: all 300ms ease-in-out;

    &:focus{
      border: 0.2rem solid ${theme.colors.secondaryColor};
    }
  }

  ${Button}{
    margin-top: 3.2rem;
    max-width: 24rem;
    width: 100%;

    &:hover{
      border: 0.2rem solid ${theme.colors.secondaryColor};
    }

    &:active{
      background:${theme.colors.primaryColor};
      color:${theme.colors.white};
      border-color:${theme.colors.white};
    }

    >img{
      margin-left: 1.4rem;
    }
  }

  @media ${theme.media.lteMedium}{
  }
  `}
`;
export const TwoInputs = styled.form`
${({ theme }) => css`
  width: 100%;
  display: flex;
  justify-content: space-between !important;

  @media ${theme.media.lteMedium}{
    flex-direction: column;
  }
  `}
`;
