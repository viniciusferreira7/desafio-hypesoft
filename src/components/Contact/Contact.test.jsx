import { renderTheme } from '../../styles/render-theme';
import { Contact } from '.';

describe('<Contact/>', () => {
  it('should match snapshot', () => {
    const { container } = renderTheme(<Contact />);
    expect(container).toMatchSnapshot();
  });
});
