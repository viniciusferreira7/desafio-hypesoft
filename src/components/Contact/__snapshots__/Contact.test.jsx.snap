// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact/> should match snapshot 1`] = `
.c2 {
  width: 15.2rem;
  height: 6.8rem;
  display: grid;
  grid-template-columns: repeat(4,1fr);
}

.c3 {
  background: #ED1CA6;
  width: 0.5rem;
  height: 0.5rem;
  margin: 0.8rem;
}

.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 10rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-top: 22.7rem;
  position: relative;
}

.c0 .c1 {
  position: absolute;
  top: 9%;
  right: 9%;
  -webkit-transform: rotate(-90deg);
  -ms-transform: rotate(-90deg);
  transform: rotate(-90deg);
  opacity: 0.3;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 1.2rem 3.2rem;
  -webkit-letter-spacing: 0.02rem;
  -moz-letter-spacing: 0.02rem;
  -ms-letter-spacing: 0.02rem;
  letter-spacing: 0.02rem;
  text-transform: uppercase;
  background: #7D00FF;
  color: #ffff;
  font-size: 1.6rem;
  border-radius: 5px;
  border: none;
  -webkit-transition: all 500ms ease-in-out;
  transition: all 500ms ease-in-out;
  cursor: pointer;
  font-weight: 600;
}

.c10:hover {
  background: #ffff;
  color: #242424;
}

.c10:hover > a {
  color: #242424;
}

.c10 > a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #ffff;
}

.c6 {
  color: #ffff;
  font-weight: 500;
  font-size: 3.2rem;
}

.c6 > strong {
  color: #ED1CA6;
}

.c6 > span {
  color: #7D00FF;
  position: relative;
  z-index: 2;
}

.c6 > span::before {
  content: '';
  opacity: 0.2;
  position: absolute;
  z-index: 1;
  top: 80%;
  left: 0;
  width: 90%;
  height: 3rem;
  background: #35016b;
}

.c17 {
  font-size: 1.8rem;
  color: #a3a0a0;
  padding: 3.2rem 0;
  line-height: 2.8rem;
}

.c4 {
  width: 100%;
  max-width: 76rem;
  padding: 4.8rem;
  background: #ffff;
  box-shadow: 0rem 1.2rem 2.4rem rgba(10,25,49,0.02);
  border-radius: 0.8rem;
}

.c4 .c5 {
  color: #242424 !important;
}

.c4 .c16 {
  width: 100%;
  background: #b1f0b8;
  border: 0.1rem solid #018f1e;
  border-radius: 2rem;
  margin-bottom: 2em;
  text-align: center;
  opacity: 0.5;
}

.c7 {
  width: 100%;
  max-width: 66.4rem;
  min-height: 28.8rem;
  margin: 2.4rem 0 3.2rem 0;
}

.c7 input {
  width: 100%;
  padding: 1.4rem 2rem;
  background: #ffff;
  border: 0.1rem solid #DFE3EB;
  color: #a3a0a0;
  border-radius: 0.4rem;
  outline: none;
  margin-bottom: 1.6rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 input:focus {
  border: 0.2rem solid #7D00FF;
}

.c7 input:first-of-type {
  margin-right: 1.6rem;
}

.c7 textarea {
  padding: 1.4rem 2.4rem;
  max-width: 66.4rem;
  width: 100%;
  height: 15.2rem;
  resize: none;
  outline: none;
  border: 0.1rem solid #DFE3EB;
  background: #ffff;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 textarea:focus {
  border: 0.2rem solid #7D00FF;
}

.c7 .c9 {
  margin-top: 3.2rem;
  max-width: 24rem;
  width: 100%;
}

.c7 .c9:hover {
  border: 0.2rem solid #7D00FF;
}

.c7 .c9:active {
  background: #242424;
  color: #ffff;
  border-color: #ffff;
}

.c7 .c9 >img {
  margin-left: 1.4rem;
}

.c8 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify !important;
  -webkit-justify-content: space-between !important;
  -ms-flex-pack: justify !important;
  justify-content: space-between !important;
}

.c15 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c11 {
  max-width: 53.6rem;
  height: 54.6rem;
  padding: 4.8rem;
  background: #ffff;
  box-shadow: 0rem 1.2rem 2.4rem rgba(10,25,49,0.02);
  border-radius: 0.8rem;
  margin-left: 2.4rem;
}

.c11 .c5 {
  color: #242424 !important;
}

.c12 {
  max-width: 44rem;
  max-height: 8rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 2.533rem 0;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c12:hover .c16 {
  -webkit-text-decoration: underline;
  text-decoration: underline;
  -webkit-text-decoration-color: #ED1CA6;
  text-decoration-color: #ED1CA6;
}

.c12:hover .c13 {
  border: 0.2rem solid #ED1CA6;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  min-width: 8rem;
  min-height: 8rem;
  margin-right: 2rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: rgba(104,79,255,0.1);
  border-radius: 4.0rem;
}

.c14 > svg {
  width: 2.667rem;
  height: 2.133rem;
  color: #ED1CA6;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  margin-top: 15rem;
}

.c18 > a {
  width: 4.3rem;
  height: 4.3rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: 0.1rem solid #DFE3EB;
  border-radius: 4.3rem;
  background: #ffff;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c18 > a:active {
  background: #7D00FF;
}

.c18 > a:active > svg {
  color: #ffff;
}

.c18 > a:hover {
  border-color: #7D00FF;
}

.c18 > a > svg {
  width: 2rem;
  height: 2rem;
  color: #242424;
}

@media (max-width:768px) {
  .c2 {
    display: none;
  }
}

@media (max-width:768px) {
  .c0 .c1 {
    display: none;
  }
}

@media (max-width:768px) {
  .c0 {
    padding: 2rem;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:768px) {
  .c4 {
    padding: 2rem;
    margin-top: 3rem;
  }
}

@media (max-width:768px) {
  .c7 input:first-of-type {
    margin-right: 0;
  }
}

@media (max-width:768px) {

}

@media (max-width:768px) {
  .c8 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:768px) {
  .c11 {
    height: auto;
    margin-top: 3rem;
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c12 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    max-height: 13rem;
    margin-bottom: 1rem;
  }
}

@media (max-width:768px) {

}

@media (max-width:768px) {

}

@media (max-width:768px) {

}

<div>
  <section
    class="c0"
    id="Contact"
  >
    <div
      class="c1 c2"
    >
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
      <div
        class="c3"
        color="#ED1CA6"
      />
    </div>
    <form
      class="c4"
    >
      <h1
        class="c5 c6"
      >
        Let me know here.
      </h1>
      <div
        class="c7"
      >
        <div
          class="c8"
        >
          <input
            id="name"
            name="name"
            placeholder="Full name"
            required=""
            type="text"
          />
          <input
            id="email"
            name="email"
            placeholder="Email address"
            required=""
            type="email"
          />
        </div>
        <input
          id="subject"
          name="subject"
          placeholder="Subjects"
          required=""
          type="subject"
        />
        <textarea
          cols="33"
          id="message"
          name="message"
          placeholder="Message"
          required=""
          rows="5"
        />
        <button
          class="c9 c10"
          sizes="1.6"
        >
          Send message 
          <img
            alt="send"
            src="assets/images/GroupPaper.png"
          />
        </button>
      </div>
    </form>
    <div
      class="c11"
    >
      <h1
        class="c5 c6"
      >
        Get In Touch
      </h1>
      <div
        class=""
      >
        <div
          class="c12"
        >
          <div
            class="c13 c14"
          >
            <svg
              aria-hidden="true"
              aria-label="email icon"
              class="c15"
              fill="currentColor"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0V0z"
                fill="none"
              />
              <path
                d="M22 6c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6zm-2 0l-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"
              />
            </svg>
          </div>
          <p
            class="c16 c17"
            color="#a3a0a0"
            sizes="1.8"
          >
            viniciuFerreira32@gmail.com
          </p>
        </div>
        <div
          class="c12"
        >
          <div
            class="c13 c14"
          >
            <svg
              aria-hidden="true"
              aria-label="telephone icon"
              class="c15"
              fill="currentColor"
              focusable="false"
              viewBox="0 0 16 16"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M3.654 1.328a.678.678 0 0 0-1.015-.063L1.605 2.3c-.483.484-.661 1.169-.45 1.77a17.568 17.568 0 0 0 4.168 6.608 17.569 17.569 0 0 0 6.608 4.168c.601.211 1.286.033 1.77-.45l1.034-1.034a.678.678 0 0 0-.063-1.015l-2.307-1.794a.678.678 0 0 0-.58-.122l-2.19.547a1.745 1.745 0 0 1-1.657-.459L5.482 8.062a1.745 1.745 0 0 1-.46-1.657l.548-2.19a.678.678 0 0 0-.122-.58L3.654 1.328zM1.884.511a1.745 1.745 0 0 1 2.612.163L6.29 2.98c.329.423.445.974.315 1.494l-.547 2.19a.678.678 0 0 0 .178.643l2.457 2.457a.678.678 0 0 0 .644.178l2.189-.547a1.745 1.745 0 0 1 1.494.315l2.306 1.794c.829.645.905 1.87.163 2.611l-1.034 1.034c-.74.74-1.846 1.065-2.877.702a18.634 18.634 0 0 1-7.01-4.42 18.634 18.634 0 0 1-4.42-7.009c-.362-1.03-.037-2.137.703-2.877L1.885.511z"
              />
            </svg>
          </div>
          <p
            class="c16 c17"
            color="#a3a0a0"
            sizes="1.8"
          >
            +55 11 95958 2011
          </p>
        </div>
      </div>
      <div
        class="c18"
      >
        <a
          href="https://github.com/viniciusferreira7"
        >
          <svg
            aria-hidden="true"
            aria-label="github icon"
            class="c15"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 16 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.012 8.012 0 0 0 16 8c0-4.42-3.58-8-8-8z"
            />
          </svg>
        </a>
        <a
          href="https://www.instagram.com/viniciusaf0/"
        >
          <svg
            aria-hidden="true"
            aria-label="instagram icon"
            class="c15"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 16 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M8 0C5.829 0 5.556.01 4.703.048 3.85.088 3.269.222 2.76.42a3.917 3.917 0 0 0-1.417.923A3.927 3.927 0 0 0 .42 2.76C.222 3.268.087 3.85.048 4.7.01 5.555 0 5.827 0 8.001c0 2.172.01 2.444.048 3.297.04.852.174 1.433.372 1.942.205.526.478.972.923 1.417.444.445.89.719 1.416.923.51.198 1.09.333 1.942.372C5.555 15.99 5.827 16 8 16s2.444-.01 3.298-.048c.851-.04 1.434-.174 1.943-.372a3.916 3.916 0 0 0 1.416-.923c.445-.445.718-.891.923-1.417.197-.509.332-1.09.372-1.942C15.99 10.445 16 10.173 16 8s-.01-2.445-.048-3.299c-.04-.851-.175-1.433-.372-1.941a3.926 3.926 0 0 0-.923-1.417A3.911 3.911 0 0 0 13.24.42c-.51-.198-1.092-.333-1.943-.372C10.443.01 10.172 0 7.998 0h.003zm-.717 1.442h.718c2.136 0 2.389.007 3.232.046.78.035 1.204.166 1.486.275.373.145.64.319.92.599.28.28.453.546.598.92.11.281.24.705.275 1.485.039.843.047 1.096.047 3.231s-.008 2.389-.047 3.232c-.035.78-.166 1.203-.275 1.485a2.47 2.47 0 0 1-.599.919c-.28.28-.546.453-.92.598-.28.11-.704.24-1.485.276-.843.038-1.096.047-3.232.047s-2.39-.009-3.233-.047c-.78-.036-1.203-.166-1.485-.276a2.478 2.478 0 0 1-.92-.598 2.48 2.48 0 0 1-.6-.92c-.109-.281-.24-.705-.275-1.485-.038-.843-.046-1.096-.046-3.233 0-2.136.008-2.388.046-3.231.036-.78.166-1.204.276-1.486.145-.373.319-.64.599-.92.28-.28.546-.453.92-.598.282-.11.705-.24 1.485-.276.738-.034 1.024-.044 2.515-.045v.002zm4.988 1.328a.96.96 0 1 0 0 1.92.96.96 0 0 0 0-1.92zm-4.27 1.122a4.109 4.109 0 1 0 0 8.217 4.109 4.109 0 0 0 0-8.217zm0 1.441a2.667 2.667 0 1 1 0 5.334 2.667 2.667 0 0 1 0-5.334z"
            />
          </svg>
        </a>
        <a
          href="https://www.linkedin.com/in/vinicius-ferreira-84b68b162/"
        >
          <svg
            aria-hidden="true"
            aria-label="linkedin icon"
            class="c15"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 16 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
            />
          </svg>
        </a>
        <a
          href="https://www.facebook.com/profile.php?id=100006780847667"
        >
          <svg
            aria-hidden="true"
            aria-label="facebook icon"
            class="c15"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 16 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
            />
          </svg>
        </a>
      </div>
    </div>
  </section>
</div>
`;
